operations/mimir/autoscaling.libsonnet

```
@@ -179,21 +211,22 @@
         {
           metric_name: '%s_cpu_hpa_%s' % [std.strReplace(name, '-', '_'), $._config.namespace],
 
-          // To scale out relatively quickly, but scale in slower, we look at the average CPU utilization per query-frontend over 5m (rolling window)
-          // and then we pick the highest value over the last 15m.
-          // Multiply by 1000 to get the result in millicores. This is due to KEDA only working with ints.
-          query: 'max_over_time(sum(rate(container_cpu_usage_seconds_total{container="%s",namespace="%s"}[5m]))[15m:]) * 1000' % [
+          query: cpuHPAQuery % [
+            name,
+            $._config.namespace,
             name,
             $._config.namespace,
           ],
```

Following the comment above where I suggested to use named placeholders, here it should be:

```suggestion
          query: cpuHPAQuery % {
            container: name,
            namespace: $._config.namespace,
          },
```

_Same comment applied below._